src/stalenessOf.js

```diff
@@ -1,17 +1,25 @@
-import { Condition, until } from 'selenium-webdriver';
+import { By, Condition, until, WebElement } from 'selenium-webdriver';
 
-import transformElementCondition from './transformElementCondition';
-
-export const checkStalenessOfFactory = stalenessOfImpl => (element) =>
+export const checkStalenessOfFactory = stalenessOfImpl => elementOrLocator =>
     async (driver) => {
+        let element = elementOrLocator;
+
+        if (elementOrLocator && !(elementOrLocator instanceof WebElement)) {
+            let locator = elementOrLocator;
+
+            if (typeof elementOrLocator === 'string') {
+                locator = By.css(elementOrLocator);
+            }
+
+            element = await driver.findElement(locator).catch(() => null);
```

Comment:
use findElements insteads and return first element. This way you won't silent error like invalid locator.